sdk/tables/azure-data-tables/samples/tables_getting_started.cpp

```diff
@@ -2,34 +2,26 @@
 // Licensed under the MIT License.
 
 #include <azure/data/tables.hpp>
+#include <azure/identity.hpp>
 
 #include <cstdio>
 #include <iostream>
 #include <stdexcept>
 #include <thread>
 
-std::string GetConnectionString()
-{
-  const static std::string ConnectionString = "";
-
-  if (!ConnectionString.empty())
-  {
-    return ConnectionString;
-  }
-  const static std::string envConnectionString = std::getenv("STANDARD_STORAGE_CONNECTION_STRING");
-  if (!envConnectionString.empty())
-  {
-    return envConnectionString;
-  }
-  throw std::runtime_error("Cannot find connection string.");
-}
-
 using namespace Azure::Data::Tables;
 const std::string TableName = "sample1";
-
+// The following environment variables must be set before running the sample.
+// * ACCOUNT_NAME: The name of the storage account.
+std::string GetAccountName() { return std::getenv("ACCOUNT_NAME"); }
+std::string const GetServiceUrl()
+{
+  return std::string{"https://" + GetAccountName() + ".table.core.windows.net/"};
+}
 int main()
 {
-  auto tableServiceClient = TableServiceClient::CreateFromConnectionString(GetConnectionString());
+  auto credential = std::make_shared<Azure::Identity::DefaultAzureCredential>();
+  auto tableServiceClient = Azure::Data::Tables::TableServiceClient(GetServiceUrl(), credential);
```

Comment:
So yea, I can see that my feedback about samples would probably apply to each sample file, please fix them all in a uniformed way, whichever way you finally decide.

Reply:
done for all, removed all usings and put the fully qualified name